cardano-cli/test/Test/Golden/Byron/SigningKeys.hs

```diff
@@ -90,7 +89,7 @@ prop_migrate_legacy_to_nonlegacy_signingkeys =
      ]
 
     eSignKey <- liftIO . runExceptT . readByronSigningKey NonLegacyByronKeyFormat
-                  $ SigningKeyFile nonLegacyKeyFp
+                  $ File nonLegacyKeyFp
```

Comment:
We can use `File` and Haskell will infer `content = OfSigningKey`.

Alternatively we can use type applications to be more specific: `File @OfSigningKey`.